update.sh

```diff
@@ -23,6 +23,12 @@ function update_node_version {
 		rm "$dockerfile.bak"
 		sed -E -i.bak 's/^(ENV YARN_VERSION ).*/\1'"$yarnVersion"'/' "$dockerfile"
 		rm "$dockerfile.bak"
+		if [ "$(echo "$version > 7.99" | bc)" == "1" ]; then
```

Comment:
One option would be to strip out the decimal and minor version and then do a regular numeric comparison:

```
version="8.0"

if [[ "${version/.*/}" -ge 8 ]]; then
  echo "True!"
else
  echo "False!"
fi
```